lib/YConverter/Converter.php

```diff
@@ -207,10 +207,15 @@ public function boot()
                 // Community Addon -> YCom
                 'replaces' => [
                     ['checkperm\(' => 'checkPerm('],
-                    ['rex_addon::get\((["']{1})community$1\)->isActivated\(\)' => 'rex_addon::get(\'ycom\')->isActivated()'],
-                    ['rex_addon::get\((["']{1})community$1\)->isAvailable\(\)' => 'rex_addon::get(\'ycom\')->isAvailable()'],
-                    ['rex_addon::get\((["']{1})community$1\)->isInstalled\(\)' => 'rex_addon::get(\'ycom\')->isInstalled()'],
+                    ['rex_addon::get\([\'\"]community[\'\"]\)->isActivated\(\)' => 'rex_addon::get(\'ycom\')->isActivated()'],
+                    ['rex_addon::get\([\'\"]community[\'\"]\)->isAvailable\(\)' => 'rex_addon::get(\'ycom\')->isAvailable()'],
+                    ['rex_addon::get\([\'\"]community[\'\"]\)->isInstalled\(\)' => 'rex_addon::get(\'ycom\')->isInstalled()'],
                     ['rex_com_auth\s*::' => 'rex_ycom_auth::'],
+                    ['com_auth_form_' => 'ycom_auth_form_'],
+                    ['com_auth_load_user' => 'ycom_auth_load_user'],
+                    ['com_auth_db' => 'ycom_auth_db'],
+                    ['com_auth_password_hash\|password\|password\|' => ''],
```

Comment:
Das wird bei den meisten nicht passen. Hier denke etwas in dieser Art müsste notiert werden:
`com_auth_password_hash\|.*?\|.*?\|?`